README.adoc

```diff
@@ -6,134 +6,23 @@ image:https://travis-ci.com/devonfw/sonar-devon4j-plugin.svg?branch=master["Buil
 
 Plugin for https://sonarqube.org[SonarQube] to validate https://github.com/devonfw/devon4j/blob/develop/documentation/coding-conventions.asciidoc#packages[devon4j architecture].
 
-== Motivation
+== Documentation
 
-With https://github.com/devonfw/devon4j/[devon4j] you can build business applications very efficiently following elaborated https://github.com/devonfw/devon4j/wiki[guidelines and best practices].
-This includes a profound https://github.com/devonfw/devon4j/blob/develop/documentation/architecture.asciidoc#architecture[architecture blueprint] that is mapped to the code via clear https://github.com/devonfw/devon4j/blob/develop/documentation/coding-conventions.asciidoc#packages[packaging conventions].
+* link:documentation/motivation.adoc[Motivation]
+* link:documentation/installation.adoc[Installation]
+* link:documentation/configuration.adoc[Configuration]
+* link:documentation/qualityprofile.adoc[devonfw Java Quality Profile]
 
-This `sonar-devon4j-plugin` provides a plugin extending https://sonarqube.org[SonarQube] with the ability to validate your Java code according to the devon4j architecture.
+== Architectural Dependency Rules
 
-== Installation
+* link:documentation/rules-architecture.adoc[Architecture Rules]
+* link:documentation/rules-layer.adoc[Layer Rules]
+* link:documentation/rules-component.adoc[Component Rules]
+* link:documentation/rules-scope.adoc[Scope Rules]
 
-If you have https://sonarqube.org[SonarQube] installed, you only need to go to its marketplace and install the latest version of this `sonar-devon4j-plugin`.
-Further you need to import the devonfw quality profiles (or activate all the rules of this plugin in your quality profile).
-For further details read the https://github.com/devonfw/sonar-devon4j-plugin/wiki/guide-sonar-qube-setup[SonarQube setup guide].
+== Implementation Rules
```

Comment:
I don't think we need to put these under a seperate header, as they are also in some way architectural rules. Just put the next four bullets as level one bullet points under the header "Architecture Rules".